README.md

```diff
@@ -22,8 +22,27 @@ modules:
       # will be auto accepted. Otherwise, all room invites are accepted.
       # Defaults to false.
       accept_invites_only_for_direct_messages: false
+
+      # (For workerised Synapse deployments)
+      # If you want to accept invites on a specific worker, specify its instance
+      # name here. Otherwise, invites will be processed on the main process.
+      #
+      # Any worker can be used.
+      #
+      #worker_to_run_on: workername1
+```
+
+
+### A note about logging
+
+Your Synapse logging configuration should have the following option set in it:
+
+```yaml
+disable_existing_loggers: False
 ```
 
+Without it, logging from this module (and potentially others) may not appear in your logs.
+
```

Comment:
What's special about modules that makes this necessary?

Reply:
Not sure, I find myself having to use this incantation is basically every Python project ever. but somehow Synapse avoids it for itself and Twisted.. :/

Reply:
I don't think synapse does avoid it... that's why it's included in https://github.com/matrix-org/synapse/blob/master/docs/sample_log_config.yaml#L84

Reply:
(the problem applies generally to python modules that are loaded before the logger is configured.)

Reply:
Ah, I see, so *some* of Synapse's logger would be getting disabled. Good to know, thanks!
Not sure why my config didn't have it.

I have to say I kind of find this flag stupid, but oh well I'm sure someone had some reason once upon a time. It may as well `dont_work_properly: False`.

Reply:
>  Not sure why my config didn't have it.

back in the very early days we didn't recommend it (see https://github.com/matrix-org/synapse/pull/6197).

>  It may as well say `dont_work_properly: False`.

agreed. "something something backwards compatibility", I expect.   TBF, it's not entirely clear that the authors of `logging.config.dictConfig` ever expected applications to pipe user configuration straight into it. Maybe we should set `disable_existing_loggers: True` in the dict before we pass it through.